assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java

```diff
@@ -497,6 +497,7 @@ public ActionReturn getGradableForSite(EntityView view , Map<String, Object> par
         data.put("submissions", submissions);
         data.put("students", students);
         data.put("groups", groups);
+        data.put("showOfficialPhoto", serverConfigurationService.getBoolean("assignment.show.official.photo", true));
```

Comment:
Is this property already documented in default.sakai.properties? just asking, I'll approve it.

Reply:
I'll added it here: https://github.com/sakaiproject/sakai/pull/6599/files#diff-2c1cd69e8a95ba38a890ca8eb93b8f02R1931.